packages/ui5-application-writer/src/data/defaults.ts

```diff
@@ -132,23 +132,44 @@ function getMinUI5Version(ui5Version: string, minUI5Version?: string) {
 }
 
 /**
- * Get the manifest descriptor version from the specified miminum UI5 version.
- * Snapshots are handled by coercion to proper versions.
+ * Get the manifest descriptor version from the specified UI5 version.
+ * Snapshots are handled by coercion to proper versions. If the version does not exist as an exact match
+ * the nearest version lower will be used.
  *
- * @param minUI5Version - the ui5 version to be used to map to the manifest descriptor version
+ * @param ui5Version - the ui5 version to be used to map to the manifest descriptor version
  * @param manifestVersion - optional manifest descriptor version to be used if provided
  * @returns - the manifest descriptor version
  */
-function getManifestVersion(minUI5Version: string, manifestVersion?: string): string {
-    const minUI5SemVer = semVer.coerce(minUI5Version)!;
-    return (
-        manifestVersion ??
-        (minUI5Version &&
-            (versionToManifestDescMapping as Record<string, string>)[
-                `${semVer.major(minUI5SemVer)}.${semVer.minor(minUI5SemVer)}`
-            ]) ??
-        UI5_DEFAULT.MANIFEST_VERSION
-    );
+function getManifestVersion(ui5Version: string, manifestVersion?: string): string {
+    const ui5SemVer = semVer.coerce(ui5Version)!;
+
+    const getClosestVersion = (ui5SemVer: semVer.SemVer) => {
```

Comment:
Minor code smell: use of the same variable name.
Doesn't cause an issue but might be confusing when reading the code.

Reply:
DOne